system-tests/dashboard/test-dashboard.js

```diff
@@ -0,0 +1,250 @@
+require("../_support/utils/ServicesUtil");
+const { Timeouts } = require("../_support/constants");
+
+// same service name as defined in app.json
+const SERVICE_NAME = "dashboard-test-service";
+
+function deleteService() {
+  cy.exec(
+    `dcos marathon app remove /${Cypress.env("TEST_UUID")}/${SERVICE_NAME}`
+  );
+  cy.visitUrl(`services/overview/%2F${Cypress.env("TEST_UUID")}`);
+  cy.get(".page-body-content table").contains(SERVICE_NAME).should("not.exist");
+}
+
+function createService() {
+  cy.exec(
+    `sed 's/dashboard-test-service/${Cypress.env("TEST_UUID")}\\/dashboard-test-service/g' /dcos-ui/system-tests/dashboard/app.json | dcos marathon app add`
+  );
+  cy.visitUrl(`services/overview/%2F${Cypress.env("TEST_UUID")}`);
+
+  cy
+    .get(".page-body-content table", {
+      timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT
+    })
+    .contains(SERVICE_NAME, { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
+    .should("exist");
+  cy
+    .get(".page-body-content table")
+    .getTableRowThatContains(SERVICE_NAME)
+    .contains("Running", { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
+    .should("exist");
+}
+
+function getAllocationElementFor(name) {
+  return cy
+    .get(".panel-header")
+    .contains(name)
+    .parents(".panel")
+    .find(".unit-label");
+}
+
+function getTaskCountElement() {
+  return cy
+    .get(".panel-header")
+    .contains("Tasks")
+    .parents(".panel")
+    .find(".unit-primary");
+}
+
+describe("Dashboard", function() {
+  describe("Create service tests", function() {
+    beforeEach(() => {
+      cy.visitUrl("dashboard");
+      const label = getAllocationElementFor("CPU Allocation");
+      label.should(function($label) {
+        expect(parseFloat($label.text())).to.equal(0);
+      });
+    });
+
+    afterEach(() => {
+      deleteService();
+      cy.window().then(win => {
+        win.location.href = "about:blank";
+      });
+    });
+
+    it("increments CPU usage when service is started", function() {
+      let label = getAllocationElementFor("CPU Allocation");
```

Comment:
In the `beforeEach()` I'm asserting this value is equal to 0, so actually I can remove this. Before all these tests there will be zero allocation.